README.md

```diff
@@ -1,26 +1,35 @@
 # Node VCX Agency
-- Implementation of [LibVCX](https://github.com/hyperledger/indy-sdk/tree/master/vcx) (V3 protocol) compatible Agency (more specifically 
-[Mediator Agency](https://github.com/hyperledger/aries-rfcs/blob/master/concepts/0046-mediators-and-relays/README.md)
-in Aries terminology).
+- Implementation of [AriesVCX](https://github.com/hyperledger/aries-vcx/) compatible 
+[Mediator Agency](https://github.com/hyperledger/aries-rfcs/blob/master/concepts/0046-mediators-and-relays/README.md).
 
 - Mediator Agency can service many users, providing them endpoint where 3rd parties can deliver encrypted messages. 
 Assuming the sending party has properly end2end encrypted the message for recipient, according to Aries connection  
 protocol, the agency itself can't decrypt content of received messages.
 
+- VCX Agency implements 2 notifications mechanism. 
+  1. Webhooks - The agency call specified url when a particular agent receives a message. This is useful when using
+     AriesVCX on server.
+  2. Longpoll - Agent owners can poll agency to check whether any new messages has arrived. This is implemented via 
+     longpoll mechanism - the server returns response only if new message has arrived, or certain amount of time has 
+     passed since the query request was received. This is to prevent overloading agency with client requests.
+     In future, longpoll mechanism might be replaced by websockets.   
+      
+
 # Repository structure
 The agency implementation is in directory [vcxagency-node](./vcxagency-node).
 
 Repository structure details:
 ```
 /
 ├── dev/               # Monorepo management scripts
-├── vcxagency-node /   # NodeJS VCX Agency implementation
-├── vcxagency-client/  # Construction of VCX Client2Agency messages
+├── vcxagency-node /   # AriesVCX mediator agency implementation in NodeJS 
+├── vcxagency-client/  # AriesVCX mediator agency client in NodeJS 
 ├── easy-indysdk/      # NodeJS idiomatic wrapper around basic IndySDK wrapper
-├-- vcx-tester/        # Agency integration tests using LibVCX
-└── node-vcx-wrapper/  # Dependency used by vcx-tester/ 
+└-- vcx-tester/        # AriesVCX integration tests using AriesVCX
 ```
 
+- Rust client for agency is to be found [here](https://github.com/hyperledger/aries-vcx/tree/master/agency_client).
```

Comment:
I would say 'Rust agency client can be found', 'to be found' to me suggests that it is somehow lost for some reason :)